Document/0x08-Testing-Tools.md

```diff
@@ -9,6 +9,7 @@ To perform security testing different tools are available in order to be able to
 * Mobisec - http://sourceforge.net/projects/mobisec/ - Mobile security testing live environment.
 * Santoku - https://santoku-linux.com/ - Santoku is an OS and can be run outside a VM as a standalone operating system.
 * Vezir Project - https://github.com/oguzhantopgul/Vezir-Project - Mobile Application Pentesting and Malware Analysis Environment.
+* ShadowOS - https://saas.hpe.com/en-us/home - Free Virtual OS for testing Android apps security.

```

Comment:
Thanks for clarifying. @bugwrangler: If you have other information to this topic, please let me know. I will close this PR and not merge it.